features/old/services/multiservice.feature

```diff
@@ -12,31 +12,29 @@ Feature: Multiservice feature
   Scenario: Can create new service setting
     Given I am logged in as provider "foo.example.com"
      And provider "foo.example.com" has "can create service" set to "true"
-    When I am on the provider dashboard page
-    Then I should see "Create Service"
-     And I should see "ID for API calls is"
-     And I should see "system name is"
+    When I am on the provider dashboard
+    Then I should see "New API"
 
   @javascript
   Scenario: Create new service
     Given I am logged in as provider "foo.example.com"
       And provider "foo.example.com" has "multiple_services" switch allowed
       And service discovery is not enabled
-    When I am on the provider dashboard page
-     And I follow "Create Service"
+    When I am on the provider dashboard
+     And I follow "New API"
      And I fill in "Name" with "Less fancy API"
-     And I press "Create Service"
+     And I press "Create"
     Then I should see "Less fancy API"
 
   Scenario: Edit service
     Given I am logged in as provider "foo.example.com"
       And I am on the edit page for service "Fancy API" of provider "foo.example.com"
     When I fill in "Name" with "Less fancy API"
      And I press "Update Service"
-     And I follow "Settings" within the side tabs
+     And I follow "Preferences" within the main menu
```

Comment:
Technically this is not correct... it should be:
```
And I follow "Integration" within the main menu
And I follow "Preferences" within the submenu
```
But I didn't get to make that work and this works because the submenu is inside the main menu xD

Reply:
I don't like this and it is what I am try to change now 😄

Reply:
Ok so with the help of Thomas, I could fix this:
![image](https://user-images.githubusercontent.com/11318903/47098106-f2e39900-d232-11e8-9a0f-0f7bab620613.png)

But this is what happens now:
![image](https://user-images.githubusercontent.com/11318903/47098125-fe36c480-d232-11e8-8646-804ee4903286.png)
And the image of the overlapping is:
![image](https://user-images.githubusercontent.com/11318903/47098261-2b837280-d233-11e8-9922-5b6e2398a32c.png)